frameworks/g-cloud-12/questions/services/serviceFeaturesSupport.yml

```diff
@@ -16,10 +16,10 @@ list_item_name: service feature
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter a service feature.
   - name: max_items_limit
     message: You can have no more than 10 features.
   - name: under_10_words
-    message: You can’t write more than 10 words for each feature.
+    message: Each feature must be 10 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?